### ISSUE TITLE
[FEATURE REQUEST]

### ISSUE THREAD

**Issue by deanZZZZZ** [flixlix/energy-flow-card-plus#124]
Display a negative secondary_info number in the grid card.
When the number is negative, it shows only 0 W

**Comment by amund7** [comment]
Yes, please add a negative feature, it is very useful to display accumulated costs. The Power Flow card recieved this as an upgrade not long ago.

**Comment by derolli1976** [comment]
I would also like that. Right now when trying to show power returned to grid as a secondary info it shows 0 when negative value is passed.
@deanZZZZZ : Could you give this issue a reasonable title?
